flutter/beagle/lib/service_locator.dart

```diff
@@ -34,19 +34,19 @@ import 'package:get_it/get_it.dart';
 
 final GetIt beagleServiceLocator = GetIt.instance;
 
-void setupServiceLocator({
-  BeagleConfig beagleConfig,
-  HttpClient httpClient,
-  Map<String, ComponentBuilder> components,
-  Storage storage,
-  bool useBeagleHeaders,
-  Map<String, ActionHandler> actions,
-  BeagleNetworkStrategy strategy,
-  Map<String, NavigationController> navigationControllers,
-  BeagleDesignSystem designSystem,
-  BeagleImageDownloader imageDownloader,
-  BeagleLogger logger,
-}) {
+void setupServiceLocator(
+    {BeagleConfig beagleConfig,
+    HttpClient httpClient,
+    Map<String, ComponentBuilder> components,
+    Storage storage,
+    bool useBeagleHeaders,
+    Map<String, ActionHandler> actions,
+    BeagleNetworkStrategy strategy,
+    Map<String, NavigationController> navigationControllers,
+    BeagleDesignSystem designSystem,
+    BeagleImageDownloader imageDownloader,
+    BeagleLogger logger,
+    Map<String, Operation> customOperations}) {
```

Comment:
Consider using trailing comma to improve future code reviews.
```suggestion
    Map<String, Operation> customOperations,
    }) {
```